errtracker/errtracker.go

```diff
@@ -67,6 +69,80 @@ var (
 	timeNow  = time.Now
 )
 
+type reportsDB struct {
+	fname string
+
+	// map of hash(dupsig) -> time-of-report
+	reported map[string]time.Time
+
+	cleanupTime time.Duration
```

Comment:
Is this the time of the last cleanup or when we wish to do the cleanup. Can you please add a comment.